doc/classes/AudioEffectDelay.xml

```diff
@@ -194,30 +197,43 @@
 	</methods>
 	<members>
 		<member name="dry" type="float" setter="set_dry" getter="get_dry">
+			Sets output percent of original sound. At 0, only delayed sounds are outputted. Value can range from 0 to 1. Default value: [code]1[/code].
```

Comment:
Remove "Sets"

Not sure how I feel about "outputted" - it's valid per the dictionary, but "output" sounds better to me...?